README.rst

```diff
@@ -37,7 +37,7 @@ authorization are editable. To do this, all data about the workflow item
 is persisted into DB. **Hence, they can be changed without touching the
 code and re-deploying your application.**
 
-There is ordering aprovments for a transition functionality in
+There are ordering improvements for a transition functionality in
```

Comment:
Changed in 372b2d562496760c3a268d7c616af28a8bf4ebb2. 

I don't think I really understand what this paragraph is trying to say. I had never heard of this word before, and it seems to be fairly obscure. None of [Wiktionary's definitions](https://en.wiktionary.org/wiki/approvement#English) seem to make sense (though neither does "improvements").

Are you trying to say there are places in the code of `django-river` where you can add your own transition functionality? What does "ordering" mean in this context?